src/components/mindset/index.js

```diff
@@ -0,0 +1,16 @@
+import {default as Mindset} from './Mindset'
+
+import {default as MindsetBody} from './MindsetBody'
+import {default as FollowUp} from './FollowUp'
+import {default as ResultsNote} from './ResultsNote'
+import {default as NextActions} from './NextActions'
+import {default as ReadMore} from './ReadMore'
+
+export {
+  MindsetBody,
+  FollowUp,
+  ResultsNote,
+  NextActions,
+  ReadMore,
+}
+export default Mindset
```

Comment:
So this is a format I've used in React a lot in the past. I can unwind it relatively easily if that's preferred. Just keeps imports a bit simpler.

Reply:
No sticklers here :) I've used the destructure as well, I think we don't need to be too strict on ourselves with it - dev preference.